setup/environment36.yml

```diff
@@ -48,15 +48,15 @@ dependencies:
 - fastcache=1.0.2=py36_1
 - flask=0.12.2=py36_0
 - flask-cors=3.0.2=py36_0
-- freetype=2.5.5=2
+- freetype=2.5.5
 - get_terminal_size=1.0.0=py36_0
 - gevent=1.2.1=py36_0
 - greenlet=0.4.12=py36_0
 - h5py=2.7.0=np112py36_0
-- hdf5=1.8.17=1
+- hdf5
```

Comment:
does `hdf5=1.8.17` work? 
Looks like it should
https://anaconda.org/conda-forge/hdf5/files

```
win-64/hdf5-1.8.17-vc9_11.tar.bz2
```